This will be fixed in 2.6.28-rc1.


ERROR: "__bad_udelay" [drivers/net/wireless/ath5k/ath5k.ko] undefined!
make[1]: *** [__modpost] Error 1

diff --git a/drivers/net/wireless/ath5k/Kconfig b/drivers/net/wireless/ath5k/Kconfig
index 75383a5..02bfd02 100644
--- a/drivers/net/wireless/ath5k/Kconfig
+++ b/drivers/net/wireless/ath5k/Kconfig
@@ -1,6 +1,6 @@
 config ATH5K
 	tristate "Atheros 5xxx wireless cards support"
-	depends on PCI && MAC80211 && WLAN_80211 && EXPERIMENTAL
+	depends on PCI && MAC80211 && WLAN_80211 && EXPERIMENTAL && (!ARM || BROKEN)
 	select MAC80211_LEDS
 	select LEDS_CLASS
 	select NEW_LEDS
